### ISSUE TITLE
PrecomputedDescriptorFromFile doesn't accept stereo smiles?

### ISSUE THREAD

**Issue by jeremycheminf** [MolecularAI/QSARtuna#3]
Hi
I'm not sure, here, but after spending some time and not understanding why my file was failing but test in data was, I used
 Function to remove stereochemistry from SMILES
```
def remove_stereo(smiles):
    mol = Chem.MolFromSmiles(smiles)
    smiles_no_stereo = Chem.MolToSmiles(mol, isomericSmiles=False)
    return smiles_no_stereo
```

Then save the files with fp column and I could run the study; otherwise I got error with descriptors can not be calculated. 
I have no idea why because other functions like "UnscaledPhyschemDescriptors" worked without any issues.

Another problem with the function, if the column we want to add is recognized as integer by pandas then it fails because the code is looking for float (I used value + 0.000001 to get around it)

**Comment by lewismervin1** [comment]
Thanks @jeremycheminf, I do not understand why the Unscaled or Scaled would fail either. Please could you share an example of the config json? I just wanted to check that the deduplication etc are the same between the runs that work and that remove those molecules?

I also just pushed https://github.com/MolecularAI/QSARtuna/commit/aacc080dfce77e92aeb02021b78866f9f6c492e1 which I hope should fix the specific issue wrt the datatype used in the precomputed descriptors

**Comment by jeremycheminf** [comment]
Hi
I managed to get this example using some smiles from the data folder and creating 2 compounds with stereo. 
Running with stereo and precomputed fails, saving file with @ removed works, even if now 2 compounds in training don't have extra info. (notebook zipped so I can upload it)
[Copy_Stereo.zip](https://github.com/MolecularAI/QSARtuna/files/14745819/Copy_Stereo.zip)

[Example_file_Qptuna.csv](https://github.com/MolecularAI/QSARtuna/files/14745814/Example_file_Qptuna.csv)

**Comment by lewismervin1** [comment]
@jeremycheminf I suspect that PR https://github.com/MolecularAI/QSARtuna/pull/18 fixes this issue as part of 3.1.1.

Please can you let me know?

Many thanks

**Comment by jeremycheminf** [comment]
Hi Lewis
Yes, the code now run with both stereo and external descriptors. Thank you!
